envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdaterTest.kt

```diff
@@ -141,45 +138,6 @@ class SnapshotUpdaterTest {
             .isEqualTo(2)
     }
 
-    @Test
-    fun `should not crash on bad snapshot generation`() {
```

Comment:
We no longer process input from the user that can throw an exception there. But maybe it would be good to test if something can throw anyway? I run into a problem while mocking this in kotlin because of Mockito's `any()` function returning null. PLAT.

Reply:
What about using https://github.com/nhaarman/mockito-kotlin or https://github.com/mockk/mockk?

Reply:
Dunno why it does not work in gradle: https://github.com/allegro/envoy-control/commit/0fa8b4f11d4c1659fc7cb2f319b4b77bac0fd689